comit/src/network/protocols/orderbook.rs

```diff
@@ -1,5 +1,5 @@
 mod order;
-mod take_order;
+pub mod take_order;
```

Comment:
Making this public exposes the internal structure of the orderbook which makes it harder to maintain a stable API of this module.

Re-exporting whatever you need from `take_order` should do the trick as well.

Reply:
So the reason why I made the module public was because there's a struct `take_order::Response` which I wanted to use just like that. Re-exporting just `Response` directly from `orderbook` seems wrong. Would it be okay to re-export as `TakeOrderResponse`? Is there a better alternative?

Reply:
> Is there a better alternative?

Can't think of anything :)